packages/forms/src/Form.js

```diff
@@ -23,10 +24,11 @@ const customWidgets = {
 	tabs: TabsWidget,
 	keyValue: KeyValueWidget,
 	multiSelectTag: MultiSelectTagWidget,
+	simpleTypeahead: SimpleTypeaheadWidget,
 };
 
 const customUiSchema = {
-	'ui:widget': ['toggle', 'tabs', 'keyValue', 'multiSelectTag'],
+	'ui:widget': ['toggle', 'tabs', 'keyValue', 'multiSelectTag', 'simpleTypeahead'],
```

Comment:
It's more powerful than a datalist, as it shows "no result" message. For me it's a Typeahead, I prefixed it with "Simple" to differenciate it from the Typehead component even though they're not in the same package.
But if you insist, ok.